.travis.yml

```diff
@@ -24,7 +24,7 @@ jobs:
       name: Installer unit tests
     - script: "docker run -v $PWD:/go/src/github.com/openshift/installer -w /go/src/github.com/openshift/installer quay.io/coreos/golang-testing go vet ./installer/..."
       name: Go vet
-    - script: "docker run -v $PWD:$PWD -w $PWD quay.io/coreos/golang-testing golint -set_exit_status installer/..."
+    - script: ./hack/go-lint.sh
```

Comment:
You've updated the script to use `"${1:-.}"`, so you should update this to pass in `installer/...`.